_data/simple-icons.json

```
@@ -12016,6 +12016,11 @@
             "hex": "000000",
             "source": "https://tidal.com"
         },
+        {
+            "title": "Tiddlywiki",
+            "hex": "111111",
+            "source": "https://talk.tiddlywiki.org/uploads/default/original/2X/d/ded4d2d4816bd6663a4ac467936d97f08af5fef8.svg"
```

```suggestion
            "source": "https://tiddlywiki.com"
```

The SVG of the logo is available on the 'loading' screen of the main site.